docs/user/communication.rst

```diff
@@ -142,9 +142,48 @@ Announcements can be created in the course communication overview by selecting t
 As soon as the announcement is created, all participants, that did not actively refrain from being notified, will receive an email containing the announcement's content.
 Additionally, announcements visually differ from normal posts and are always displayed on top of the communication overview.
 
+Notifications
+-------------
+
+Artemis supports customizable web and email notifications. Users can enable and disable different notification types.
+
+Web Notifications
+^^^^^^^^^^^^^^^^^
+
+The web notifications can be found on the top right of the page by clicking on the bell icon.
+A red number shows the number of new messages.
```

Comment:
number twice
```suggestion
A red number shows the amount of new messages.
```

Reply:
I think number is actually correct here, as amount is used for uncountable elements (‘Anzahl’ vs ‘Menge’ in German).